README.md

```diff
@@ -22,11 +22,12 @@ Some use cases include:
 
 ## Usage
 
-* Install the package
+Install the package
+
```

Comment:
Removed the list format (only two items). The list also avoids the installation command of being rendered properly.